gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/ReminderSettingFlowTriggerLeaseArbiter.java

```diff
@@ -332,4 +303,17 @@ protected static long getUTCTimeFromDelayPeriod(long delayPeriodMillis) {
     Date date = Date.from(localDateTime.atZone(ZoneId.of("UTC")).toInstant());
     return GobblinServiceJobScheduler.utcDateAsUTCEpochMillis(date);
   }
+
+  @Override
+  public LeaseAttemptStatus tryAcquireLease(DagActionStore.DagAction dagAction, long eventTimeMillis,
+      boolean isReminderEvent, boolean adoptConsensusFlowExecutionId)
+      throws IOException {
+    return this.decoratedLeaseArbiter.tryAcquireLease(dagAction, eventTimeMillis, isReminderEvent, adoptConsensusFlowExecutionId);
+  }
+
+  @Override
+  public boolean recordLeaseSuccess(LeaseObtainedStatus status)
+      throws IOException {
+    return this.decoratedLeaseArbiter.recordLeaseSuccess(status);
+  }
```

Comment:
this abstraction seems ill-suited to the Decorator pattern, as there's no decoration going on